src/main/java/robotbuilder/exporters/ExportFile.java

```diff
@@ -76,6 +86,10 @@ void backup(GenericExporter exporter) throws IOException {
         }
     }
 
+    private boolean equals(String s1, String s2) {
+        return (s1 == null) ? (s2 == null) : (s1.equals(s2));
+    }
```

Comment:
There's already `Objects.equals` that does this. Why not use that?
https://docs.oracle.com/javase/7/docs/api/java/util/Objects.html#equals(java.lang.Object,%20java.lang.Object)

Reply:
OK, fixed.